### DISCUSSION TITLE
A "mode" where one could visualize the internal operation/execution of a generator without it producing any output.

### DISCUSSION THREAD

**Discussion by AceTheWiz** [statiqdev/Discussions#9] [Ideas]
### My Scenario
I have created a basic generator but I have added settings to modify the input and output directories and file patterns using the globbing syntax.

### My Problem
I do not know which files are going to be ingested and which are excluded. This is probably because I am not an expert on globbing syntax.

### The Proposal
A feature (possibly a command line option) where I would be able to specify a "no output" mode where my generator would only spit out the logging messages so I can visualize which folders and files are going to be processed.

I have seen this feature in build tools where one can visualize the execution without actually producing outputs.

**Comment by StevenTCramer** [comment]
There now is a glob testing command.

`dotnet run -- glob eval <pattern> <path>`

docs for it are https://www.statiq.dev/guide/execution/files
